Add lovelace resource mode (#43318)
home-assistant/home-assistant.github.io: source/dashboards/dashboards.markdown

      title: Admin
      icon: mdi:tools
      show_in_sidebar: true
      require_admin: true
      filename: admin.yaml
```

{% configuration dashboards %}
resource_mode:
  required: false
  description: "Controls how resources are loaded. Set to `yaml` to load resources from the `resources` key in YAML configuration. Set to `storage` to manage resources through the UI."
  type: string
  default: storage
resources:
  required: false

>   description: "List of resources that should be loaded. Requires `resource_mode: yaml` to take effect. If you change anything here, click the three dots {% icon "mdi:dots-vertical" %} menu (top-right) and click **Reload resources** to pick up changes without restarting Home Assistant. You can also call `lovelace.reload_resources` action directly."

Reviewer: ```suggestion
  description: "List of resources that should be loaded. Requires `resource_mode: yaml` to take effect. After changing the YAML configuration, select the three dots {% icon "mdi:dots-vertical" %} menu (top-right) and select **Reload resources** to pick up changes without restarting Home Assistant. You can also call the `lovelace.reload_resources` action directly."
```

>   description: "List of resources that should be loaded. Requires `resource_mode: yaml` to take effect. If you change anything here, click the three dots {% icon "mdi:dots-vertical" %} menu (top-right) and click **Reload resources** to pick up changes without restarting Home Assistant. You can also call `lovelace.reload_resources` action directly."

Author: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/home-assistant/home-assistant.io/pull/43318#discussion_r2741823909)

      type: string
dashboards:
  required: false

>   description: "Additional YAML dashboards. The key is used for the URL and should contain a hyphen (`-`), except for `lovelace` which is allowed for backward compatibility."

Reviewer: ```suggestion
  description: "Additional YAML dashboards. The key is used for the URL and should contain a hyphen (`-`), except for `lovelace`, which is allowed for backward compatibility."
```
